x-pack/metricbeat/module/airflow/statsd/_meta/data.json

```diff
@@ -2,13 +2,13 @@
     "@timestamp": "2017-10-12T08:05:34.853Z",
     "airflow": {
         "dag_duration": {
-            "15m_rate": 0.2,
-            "1m_rate": 0.2,
-            "5m_rate": 0.2,
+            "15m_rate": 0,
+            "1m_rate": 0,
+            "5m_rate": 0,
             "count": 1,
             "max": 200,
             "mean": 200,
-            "mean_rate": 0.2222490946071946,
+            "mean_rate": 38960.532980091164,
```

Comment:
@ritalwar , i hope this value is generated one. The mean_rate value doesn't sound justifiable compared to the max , mean and median values. Please validate

Reply:
Yes, Agi, this is generated [here](https://github.com/ritalwar/beats/blob/4fc3100f11f5e164f1819b45c59f9da882d94f97/x-pack/metricbeat/module/airflow/statsd/data_test.go#L52) using the test data supplied [here](https://github.com/ritalwar/beats/blob/4fc3100f11f5e164f1819b45c59f9da882d94f97/x-pack/metricbeat/module/airflow/statsd/data_test.go#L79).

Reply:
Also, it is the meter's mean rate of events per second at the time the snapshot was taken.